.gitignore

```diff
@@ -19,3 +19,8 @@ test/*/results/
 LearningTest/*/*/*/results/
 LearningTest/*/*/*/results.ref/time.log
 
+# LearningTest 
+results/
+__pycache__/
+comparisonResults.log
```

Comment:
-  put the learning test exceptions as I tried to above (the path would be `test/LearningTest/*/*/*/results`, etc). We could have another "results" directory that we don't want to ignore.
- `__pycache__` should be on its own section (python)

Reply:
Hello, just this change and it will LGTM